### PR TITLE
Fix initial migration filename

### PR DESCRIPTION
**Pull request by BrunoBernardino**
I suppose it used to be `1_initial_deployment.js` but now it's `1_initial_migration.js` and there are other parts in this doc where it was updated. Probably just missed this one.

### CODE REVIEW

File: src/docs/truffle/quickstart.md [hunk 1]
```diff
@@ -353,4 +353,4 @@ Interact with the contract using the console in the following ways:
 
 ## Continue learning
 
-This quickstart showed you the basics of the Truffle project lifecycle, but there is much more to learn. Please continue on with the rest of our [documentation](/docs) and especially our [tutorials](/tutorials) to learn more.
\ No newline at end of file
+This quickstart showed you the basics of the Truffle project lifecycle, but there is much more to learn. Please continue on with the rest of our [documentation](/docs) and especially our [tutorials](/tutorials) to learn more.
```
[the remark this answers is not in the document]
Comment (CruzMolina) [reply src/docs/truffle/quickstart.md]: Looks like a whitespace change.

[the remark this answers is not in the document]
Comment (BrunoBernardino) [reply src/docs/truffle/quickstart.md]: Want me to revert this bit?

**Comment by BrunoBernardino** [comment]
And is there anything else you want me to do in order to merge this?

File: src/docs/truffle/quickstart.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (CruzMolina) [reply src/docs/truffle/quickstart.md]: @BrunoBernardino can you revert the whitespace change? Will merge after. Good catch! 😎

**Comment by CruzMolina** [comment]
Ah, I'm seeing "No newline at end of file" in the diff.

**Comment by BrunoBernardino** [comment]
Good point! I thought that was it initially, but adding a new empty line didn’t make that diff go away.

That “no new line” is part of unchanged diff, I believe (the old version also had no new line).

File: src/docs/truffle/quickstart.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (CruzMolina) [reply src/docs/truffle/quickstart.md]: Maybe you're saving with a new line added to the end of file?

**Comment by BrunoBernardino** [comment]
I'm ok closing this and you can make the change I suggested here @CruzMolina . I'd be curious to see if you also get that whitespace change.

File: src/docs/truffle/quickstart.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (CruzMolina) [reply src/docs/truffle/quickstart.md]: For reference, I'm seeing on github only 356 lines in the overview file on `master` while your branch has 357.

[the remark this answers is not in the document]
Comment (BrunoBernardino) [reply src/docs/truffle/quickstart.md]: Ohh. I see that, good point! But locally, I only have 356:

```sh
wc -l quickstart.md
     356 quickstart.md
```

[the remark this answers is not in the document]
Comment (CruzMolina) [reply src/docs/truffle/quickstart.md]: maybe try a direct fork on github, make the fix, and then a new PR?

[the remark this answers is not in the document]
Comment (BrunoBernardino) [reply src/docs/truffle/quickstart.md]: Thanks, I've tried that initially, then a local fork via `git`. All same result 🤷‍♂️

[the remark this answers is not in the document]
Comment (CruzMolina) [reply src/docs/truffle/quickstart.md]: Sorry for the delay! I ran into the same issues trying to make the fix myself, so I'm just going to move forward with merging this.

Thanks again and sorry it's been such a pain!

[the remark this answers is not in the document]
Comment (BrunoBernardino) [reply src/docs/truffle/quickstart.md]: No worries, and thank you!

**Pull request merged**